vitals/SwitchLanguage.java

```diff
@@ -0,0 +1,78 @@
+package vitals;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class SwitchLanguage {
+	
+	public static String language;
+	static List<String> availableLanguage = Arrays.asList("EN", "DE");
+	
+	public static boolean setLanguageInSystem(String lan) {
+		if(availableLanguage.contains(lan)){
+	    	language=lan;
+	        return true;
+	    }
+		else {
+			//language is not supported!!!
+		return false;
+		}
+	}
+	
+	
+	public static void setMessage(String messagePrint)
+    {
+      System.out.println(messagePrint);
+    }
+    
+    public static void maxLimit(String parameter, float MaximumLimit)
+    {
+    	
+    	if (language.equals("DE") ) {
+    		String displayMessage =parameter + " " + BatteryConstants.HIGH_THRESHOLD_DE + " " + MaximumLimit;
+        	setMessage(displayMessage);
+    	}

```

Comment:
min , max ,low , high function contains duplicate string format and language check code ... if we introduce new language ? how many code blocks in this program subject for change ?

Reply:
Logic is now handled with Locale and ResourceBundle for handling extra languages